connectors/protocol/connectors.py

```diff
@@ -762,27 +765,32 @@ async def prepare(self, config):
                 config["sources"][configured_service_type], configured_service_type
             )
 
-            return
-
         doc = {}
+        fqn = config["sources"][configured_service_type]
```

Comment:
What does `fqn` stand for? I see it came from below, I'm just curious.

Reply:
[Fully qualified name](https://en.wikipedia.org/wiki/Fully_qualified_name) 

I think this has confused many people, and we should consider renaming it 😅